rust/codelist-rs/src/code_entry.rs

```diff
@@ -16,8 +16,8 @@ use serde::{Deserialize, Serialize};
 /// * `term` - The term
 #[derive(Debug, Serialize, Deserialize, PartialEq, Clone, Eq, Hash)]
 pub struct CodeEntry {
-    code: String,
-    term: String,
+    pub code: String,
+    pub term: String,
```

Comment:
For the future, we might want to change new so that it will accept a numnber (int) which is the format of SNOMED and it converts it to string